test/module/irohad/synchronizer/synchronizer_test.cpp

```diff
@@ -78,7 +78,9 @@ TEST_F(SynchronizerTest, ValidWhenSingleCommitSynchronized) {
 
   EXPECT_CALL(*mutable_factory, commit_(_)).Times(1);
 
-  EXPECT_CALL(*chain_validator, validateBlock(test_block, _))
+// TODO: 14-02-2018 Alexey Chernyshov replace with expected argument after relocation to
+// shared_model https://soramitsu.atlassian.net/browse/IR-903
+  EXPECT_CALL(*chain_validator, validateBlock(/* testing::Ref(new_test_block) */ _, _))
```

Comment:
What is the purpose of commented code here?

Reply:
uncomment /* testing::Ref(new_test_block) */ after relocation to the new model (IR-903?)